aspnetcore/security/authorization/secure-data/samples/final2/Startup.cs

```diff
@@ -38,28 +38,28 @@ public void ConfigureServices(IServiceCollection services)
                 .AddEntityFrameworkStores<ApplicationDbContext>()
                 .AddDefaultTokenProviders();
 
-            services.AddMvc()
-                .AddRazorPagesOptions(options =>
-                {
-                    options.Conventions.AuthorizeFolder("/Account/Manage");
-                    options.Conventions.AuthorizePage("/Account/Logout");
-                });
-
-            services.AddSingleton<IEmailSender, EmailSender>();
-
-            var skipSSL = Configuration.GetValue<bool>("LocalTest:skipSSL");
+            var skipHTTPS = Configuration.GetValue<bool>("LocalTest:skipHTTPS");
             // requires using Microsoft.AspNetCore.Mvc;
             services.Configure<MvcOptions>(options =>
             {
-                // Set LocalTest:skipSSL to true to skip SSL requrement in 
+                // Set LocalTest:skipHTTPS to true to skip SSL requrement in 
                 // debug mode. This is useful when not using Visual Studio.
-                if (Environment.IsDevelopment() && !skipSSL)
+                if (Environment.IsDevelopment() && !skipHTTPS)
                 {
                     options.Filters.Add(new RequireHttpsAttribute());
```

Comment:
We don't recommend using RequireHttpsAttribute and I'm going to suggest deprecating it.

Reply:
@javiercn  Should I replace it with `AddRedirectToHttps`? Don't we need to enforce HTTPS?